### PR TITLE
Build Times

### PR DESCRIPTION
**Pull request by sme23**
Takes Stan's build times and makes them a flag, `--build-times`.

### CODE REVIEW

File: ColorzCore/Program.cs [hunk 1]
```diff
@@ -259,11 +274,156 @@ static int Main(string[] args)
...
             inStream.Close();
             output.Close();
             errorStream.Close();
 
             return success ? EXIT_SUCCESS : EXIT_FAILURE;
+
+
+        }
+
+        public class ExecTimer
```
Comment (Crazycolorz5) [inline ColorzCore/Program.cs:307]: Move to its own file

File: ColorzCore/EAInterpreter.cs [hunk 2]
```diff
@@ -73,6 +73,9 @@ public EAInterpreter(IOutput output, string game, string rawsFolder, string raws
         public bool Interpret()
         {
             Tokenizer t = new Tokenizer();
+            //ROM myROM = new ROM(fout);
```
Comment (Crazycolorz5) [inline ColorzCore/EAInterpreter.cs:76]: Remove unnecessary comment

File: ColorzCore/Program.cs [hunk 3]
```diff
@@ -8,6 +9,10 @@ namespace ColorzCore
     class Program
     {
         public static bool Debug = false;
+
+
+        public static ExecTimer Timer = null;
```
Comment (Crazycolorz5) [inline ColorzCore/Program.cs:14]: Instead of a static variable, use the Singleton pattern on ExecTimer. 
nulls bad.

Comment (sme23) [reply ColorzCore/Program.cs]: ~~i'm bad and don't understand what a singleton is in this context, but did my last commit fix this~~

Comment (Crazycolorz5) [reply ColorzCore/Program.cs]: Lemme pull it and take care of it

Comment (Crazycolorz5) [reply ColorzCore/Program.cs]: https://www.oodesign.com/singleton-pattern.html
Here's a link for reference. It's an Object Oriented design pattern, and I think this page talks about its motivation pretty well (and you should be able to see why it applies here).
(Note: I only got this page off Googling, no endorsement or anything in particular meant)

**Pull request merged**